src/Lumi/Components2/QRCode.purs

```diff
@@ -36,7 +36,7 @@ derive instance ntUseQRCode :: Newtype (UseQRCode hooks) _
 data ErrorCorrectLevel
   = ECLLow
   | ECLMedium
-  | ECLQuality
+  | ECLQuartile
```

Comment:
Bad assumption on my part when I initially wrote this, it's "Q" as in "Quartile"